### PR TITLE
Added minimal WPF designer support

### PR DESCRIPTION
**Pull request by merceyz**
- Fixed WPF designer support

### CODE REVIEW

File: CefSharp.Wpf/ChromiumWebBrowser.cs [hunk 1]
```diff
@@ -1473,7 +1502,7 @@ private void RemoveSourceHook()
         /// <returns>bool to indicate if browser was created. If the browser has already been created then this will return false.</returns>
         protected virtual bool CreateOffscreenBrowser(Size size)
         {
-            if (browserCreated || System.ComponentModel.DesignerProperties.GetIsInDesignMode(this) || size.IsEmpty || size.Equals(new Size(0, 0)))
+            if (browserCreated || size.IsEmpty || size.Equals(new Size(0, 0)))
```
Comment (amaitland) [inline CefSharp.Wpf/ChromiumWebBrowser.cs]: Is there a reason we're no longer checking design mode here?

Comment (merceyz) [inline CefSharp.Wpf/ChromiumWebBrowser.cs]: `CreateOffscreenBrowser` gets called from `OnActualSizeChanged` which isn't registered when in design mode.

Comment (amaitland) [inline CefSharp.Wpf/ChromiumWebBrowser.cs]: Makes sense 👍

**Comment by amaitland** [comment]
All looks pretty straight forward 👍

**Pull request merged**